source/validate_types.cpp

```diff
@@ -205,11 +205,9 @@ bool IsInstructionInLayoutSection(ModuleLayoutSection layout, SpvOp op) {
 }
 
 // NOTE: We are using vector to map the ID of the capability to its
```

Comment:
I agree with @dekimir's advice in general that the blurb for a construct should describe the intrinsic meaning of the object.   Don't comment on the use.  So the text about "We are using the vector to map ...." should really be a comment at the declaration of module_capabilities_.  There is where you describe how the vector is indexed.